docs/guides/logging.md

```diff
@@ -28,7 +28,7 @@ Writing a log is as simple as `#!cpp CROW_LOG_<LOG LEVEL> << "Hello";` (replace&
 
 !!! note
 
-    Log times are reported in GMT timezone by default. This is because HTTP requires all reported times for requests and responses to be in GMT. This can be changed by using the macro `CROW_USE_LOCALTIMEZONE` which will set **only the log timezone** to the server's local timezone.
+    Log times are reported in UTC timezone by default. This is because HTTP requires all reported times for requests and responses to be in UTC. This can be changed by using the macro `CROW_USE_LOCALTIMEZONE` which will set **only the log timezone** to the server's local timezone.
```

Comment:
I would suggest this be kept as GMT, since in headers themselves GMT is mentioned instead of UTC, I don't believe there's a difference between GMT and UTC anyway, is there?

Reply:
You're right, I thought that UTC was more official/universal and had replaced GMT, but it seems like both are still commonly used interchangeably (though some people have strong feelings about using one over the other)